jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildTarMojo.java

```diff
@@ -65,6 +68,12 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     MavenProjectProperties projectProperties =
         MavenProjectProperties.getForProject(
             getProject(), getSession(), getLog(), tempDirectoryProvider);
+    UpdateChecker updateChecker =
+        UpdateChecker.checkForUpdate(
+            projectProperties.isOffline() || !getLog().isInfoEnabled(),
+            projectProperties.getVersion(),
```

Comment:
I believe this returns the project version, like `<version>0.0.1-SNAPSHOT</version>` in `pom.xml`. We have `ProjectInfo.VERSION` (which works only when the code is packaged as a JAR).